frontend/i18n.js

```diff
@@ -1,6 +1,17 @@
-{
+/*!
+ * This file is part of the Stooa codebase.
+ *
+ * (c) 2020 - present Runroom SL
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+const DEFAULT_LOCALE = 'en';
+
+module.exports = {
```

Comment:
If we move to js file, we should try to use single quotes or no quotes if not needed instead of the previous double quotes (needed because of the previous json format)